### ISSUE TITLE
[HOLD for payment 2023-04-26] [$1000] Wrong transition in Login page when click Login menu

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#16981]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Open app and do not log in (sign out if already log in)
2. Navigate to Log in page
3. Scroll down to Footer
4. Click Log in
5. See that the page is re-directed, i.e. the transition is left push

## Expected Result: 
The page should scroll up to the Phone or email input since this is the same page
## Actual Result: 
The page is re-directed and re-direction transition occurs

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native
- [ ] Android / Chrome
- [x] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.2.95-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/43996225/230212633-7e1dd419-8b17-47fe-a60a-407b8e3d315e.MP4

https://user-images.githubusercontent.com/43996225/230212649-fff73250-e907-49cc-8c61-b0cf2e8ac47f.mp4

**Expensify/Expensify Issue URL:**
**Issue reported by:** @tienifr 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1680668363755569

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0130b29416abee802b</li>
        <li>Upwork Job ID: 1644034500219781120</li>
        <li>Last Price Increase: 2023-04-06</li>
    </ul>
</details>

**Comment by MelvinBot** [comment]
Triggered auto assignment to @puneetlath (`Bug`), see https://stackoverflow.com/c/expensify/questions/14418 for more details.

**Comment by MelvinBot** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [ ] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [ ] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [ ] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [ ] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [ ] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by MelvinBot** [comment]
Job added to Upwork: https://www.upwork.com/jobs/~0130b29416abee802b

**Comment by MelvinBot** [comment]
Current assignee @puneetlath is eligible for the External assigner, not assigning anyone new.

**Comment by MelvinBot** [comment]
Triggered auto assignment to Contributor-plus team member for initial proposal review - @rushatgabhane (`External`)

**Comment by MelvinBot** [comment]
Current assignee @puneetlath is eligible for the External assigner, not assigning anyone new.

**Comment by rushatgabhane** [comment]
@allroundexperts sounds good, sure let's animate as well.

@puneetlath I agree with the problem, root cause and the [solution](https://github.com/Expensify/App/issues/16981#issuecomment-1499431488).
C+ reviewed 🎀 👀 🎀

**Comment by puneetlath** [comment]
Sounds good to me!

**Comment by MelvinBot** [comment]
📣 @allroundexperts You have been assigned to this job by @puneetlath!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) 📖

**Comment by hellohublot** [comment]
Hello, should we consider this situation ?

https://github.com/Expensify/App/issues/16357#issuecomment-1495240483

I don't know if that my [solution A](https://github.com/Expensify/App/issues/16357#issuecomment-1493072701) will be accepted because he told me it's another issue

If he can't accept my proposal, I hope you can consider this case or add this code, because it can also solve [that issue](https://github.com/Expensify/App/issues/16357)

finally we can close the two issues together

**Comment by bernhardoj** [comment]
I don't think scrolling would be a solution here https://github.com/Expensify/App/issues/16357#issuecomment-1494945055

**Comment by MelvinBot** [comment]
@puneetlath, @rushatgabhane, @allroundexperts Whoops! This issue is 2 days overdue. Let's get this updated quick!

**Comment by puneetlath** [comment]
This is on staging.

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.3.1-3 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/17119


If no regressions arise, payment will be issued on 2023-04-26. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by MelvinBot** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@rushatgabhane] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@rushatgabhane] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@rushatgabhane] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@puneetlath] Determine if we should create a regression test for this bug.
- [ ] [@rushatgabhane] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@puneetlath] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by puneetlath** [comment]
@tienifr @allroundexperts @rushatgabhane sent you all contracts.

**Comment by Santhosh-Sellavel** [comment]
The PR for this issue is causing a Regression/behaviour changes details [here](https://github.com/Expensify/App/pull/17119#issuecomment-1517015392)

**Comment by Santhosh-Sellavel** [comment]
@rushatgabhane will follow up here

**Comment by puneetlath** [comment]
@rushatgabhane can you help me understand the status of this issue?

**Comment by puneetlath** [comment]
@rushat bump!

**Comment by puneetlath** [comment]
Ah ok, cool! In that case, @rushatgabhane sounds like we just need to do the checklist and we'll be done here.

**Comment by rushat** [comment]
Hi all,

You got the wrong Rushat.

**Comment by MelvinBot** [comment]
📣 @rushat! 📣

Hey, it seems we don’t have your contributor details yet! You'll only have to do this once, and this is how we'll hire you on Upwork.
Please follow these steps:

1. Get the email address used to login to your Expensify account. If you don't already have an Expensify account, create one [here](https://new.expensify.com/). If you have multiple accounts (e.g. one for testing), please use your main account email.
2. Get the link to your Upwork profile. It's necessary because we only pay via Upwork. You can access it by logging in, and then clicking on your name. It'll look like [this](https://www.upwork.com/freelancers/~01ca8ad8ce36b9eceb). If you don't already have an account, sign up for one [here](https://www.upwork.com/nx/signup/?dest=home).
3. Copy the format below and paste it in a comment on this issue. Replace the placeholder text with your actual details.

![Screen Shot 2022-11-16 at 4 42 54 PM](https://user-images.githubusercontent.com/30609178/231263719-a8a3b7d3-36a0-4d7f-8ad4-cebe1f560fa4.png)

Format:
```
Contributor details
Your Expensify account email: <REPLACE EMAIL HERE>
Upwork Profile Link: <REPLACE LINK HERE>
```

**Comment by puneetlath** [comment]
Whoops, sorry about that!

**Comment by puneetlath** [comment]
Thanks Rushat. Can you accept the job offer on Upwork as well so I can get you paid out?

**Comment by puneetlath** [comment]
Just waiting for @rushatgabhane to accept the Upwork offer.

**Comment by puneetlath** [comment]
All paid!
